.rubocop.yml

```diff
@@ -0,0 +1,124 @@
+AllCops:
```

Comment:
I think we should follow https://github.com/github/rubocop-github and create a gem `rubocop-rails` or something and use it all over the rails repositories with `inherit_gem`